src/jenesis/contracts/deploy.py

```diff
@@ -32,11 +32,11 @@ def insert(data: Union[Dict, List], contract_name: str, address: str):
 def insert_address(contract_address_names: List[str], deployment: Deployment, profile: Profile) -> dict:
```

Comment:
Is `contract_address_names` meant to be a set or a list? I think it is getting a set from the code below. Both are iterable, so it may not matter in this case, but just wanted to check to make sure.

Reply:
Good catch! Yeah, it is a set indeed. Not sure why it wasn't causing any trouble. Ill convert "contract_address_names" to a list before if that's ok. Another option would be to change the type in insert_address function to set()

Reply:
The point of the set is to remove repeated entries in the list of contract address names, right? And probably the recursive function doesn't work for sets, so yeah I think converting from list to set and back to list sounds reasonable.

Reply:
These are exactly the kind of bugs that are not really bugs but could cause problems down the line that type hinting is great for!